tests/tests_pytorch/callbacks/progress/test_rich_progress_bar.py

```diff
@@ -401,3 +401,16 @@ def test_step(self, batch, batch_idx):
 
     trainer.test(model, verbose=False)
     assert pbar.calls["test"] == []
+
+
+@RunIf(rich=True)
+def test_rich_progress_bar_padding(tmpdir):
+
+    progress_bar = RichProgressBar()
+    model = BoringModel()
+
+    trainer = Trainer(default_root_dir=tmpdir, callbacks=progress_bar, fast_dev_run=True)
+    trainer.fit(model)
```

Comment:
It's not required that we `fit` a model for this test. It doesn't even need to have `RunIf(rich=True)` if instead we unit-test the `_get_train_description` method

Reply:
But it would require class attributes like progress, etc to figure out the description for the progress bar.

Reply:
That's fine. See for example:

```py
def test_rich_progress_bar_padding():
    progress_bar = RichProgressBar()
    trainer = Mock()
    trainer.max_epochs = 1
    progress_bar._trainer = trainer

    train_description = progress_bar._get_train_description(current_epoch=0)
    assert "Epoch 0/0" in train_description
    assert len(progress_bar.validation_description) == len(train_description)
```

Reply:
Done!